resources/default_pipeline_environment.yml

```diff
@@ -115,8 +115,17 @@ steps:
       opensourceConfiguration: ''
       pipelineConfigAndTests: ''
       securityDescriptor: ''
-      'snyk configuration': ''
       tests: ''
+  snykExecute:
+    buildDescriptorFile: './package.json'
+    excludeMtaModules: []
```

Comment:
@CCFenner shouldn't we name this simply name this parameter exclude?

Reply:
this only is effective on `scanType: 'mta'`, therefore I changed the name to be more specific

Reply:
Is it working on modules / directories or on paths to descriptor files since the name could then be misleading a bit.

Reply:
You're right, it's working on descriptor files and therefore misleading.